### PR TITLE
Update to arch + forge port + keybinds

### PR DESCRIPTION
**Pull request by ferriarnus**
So this is a rather big one. First up I've turned it into an arch project. Unfortunately the client <-> server split wasn't being nice so I had to remove that. After that I added support for forge. Unfortunately the forge config screen isn't available in game (only in the main menu), so I've added a key bind instead (and to fabric as well). 

P.S. On a personal note I'm not a fan of default on analytics in a mod. I would rather have a screen on first boot asking users, or defaulting to off.

### CODE REVIEW

File: forge/src/main/resources/pack.mcmeta [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "pack": {
+    "description": "SoundsBeGone",
+    "pack_format": 13
```
Comment (meza) [inline forge/src/main/resources/pack.mcmeta:4]: This might need to be 15 for 1.20.1

File: forge/src/main/resources/examplemod.mixins.json
Comment (meza) [inline forge/src/main/resources/examplemod.mixins.json:1]: can you rename this so that it's not "examplemod"? :)

**Comment by meza** [comment]
Incredible PR, I really appreciate the help!

File: forge/src/main/resources/examplemod.mixins.json
Comment (ferriarnus) [reply forge/src/main/resources/examplemod.mixins.json:1]: Oh the file isn't even used, so it can be removed

**Pull request merged**